src/backgrounds.jl

```diff
@@ -0,0 +1,216 @@
+"""
+
+`draw_grid(video::Video, action::Action, frame::Int; direction::Symbol = :TR, line_gap::Int = 25)`
+
+Draws an oriented grid on the given frame of a Video.
+
+# Arguments
+- `direction::Symbol`: Where grid animation finishes. Default: `:TR` Available Orientations:
+  - `:TR` - Animation finishes in the **T**op **R**ight corner of the frame.
+  - `:TL` - Animation finishes in the **T**op **L**eft corner of the frame.
+  - `:BR` - Animation finishes in the **B**ottom **R**ight corner of the frame.
+  - `:BL` - Animation finishes in the **B**ottom **L**eft corner of the frame.
+- `line_gap::Int`: How many pixels between each line. Default: `25`
+
+# Example
+This example will produce a grid animation that finishes in the top left corner of the animation. One will need to define their own path for `tempdirectory` and `pathname`.
+
+```
+using Javis
+using Luxor
+
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+
+video = Video(500, 500)
+javis(
+      video,
+      [
+          Action(1:100, ground),
+    	  Action(1:100, :line, (video, action, frame)-> draw_grid(video, action, frame, direction = :TL, line_gap = 25)),
+      ],
+      tempdirectory = "/tmp-directory",
+      creategif = true,
+      pathname = "/tmp-directory/grid_animation.gif"
+     )
+
+```
+
+"""
+function draw_grid(
+    video::Video,
+    action::Action,
+    frame::Int;
+    direction::Symbol = :TR,
+    line_gap::Int = 25,
+)
+
+    min_width = div(video.width, -2, RoundDown)
+    max_width = div(video.width, 2, RoundUp)
+
+    min_height = div(video.height, -2, RoundDown)
+    max_height = div(video.height, 2, RoundUp)
+
+    step = (frame - first(action.frames)) / (length(action.frames) - 1)
+
+    if direction == :BR
+
+        for y_point = min_width:line_gap:max_width
```

Comment:
In general I would prefer using `Val` here as it makes it more extendable.
Maybe someone wants to animate from the origin outwards. At the moment he/she would need to make a PR or use a fork to add another `if` in the future it should be possible to simply create a new function like:
```
function Javis.draw_grid(video, action, frame, ::Val{outwards}; line_gap=25)
```
and call `draw_grid(video, action, frame; direction=:outwards, line_gap=25)`
which dispatches to the new function which might live outside of Javis.
Hope that makes sense. Doesn't need to be done now but is helpful for another PR if you want to work on it.

Reply:
That makes sense and was actually something I was thinking about doing, but didn't feel it appropriate for this initial PR. I agree, let's add it in another PR in the future (ping #39).

I am going to resolve this for now.